packages/esm-patient-allergies-app/src/allergies/allergies-form/allergy-form.component.tsx

```diff
@@ -82,23 +89,63 @@ function AllergyForm(props: DefaultWorkspaceProps) {
   const [isDisabled, setIsDisabled] = useState(true);
   const { mutate } = useAllergies(patientUuid);
 
+  const getDefaultAllergy = (allergy: Allergy, formContext) => {
```

Comment:
Can we break more of this logic down into separate functions for readability? Maybe we could extract a function for getting the allergic reactions, and another for getting the severity UUID for a given reactionSeverity.